fvm/CHANGELOG.md

```diff
@@ -4,32 +4,41 @@ Changes to the reference FVM implementation.
 
 ## [Unreleased]
 
-Default the concurrency of the `ThreadedExecutor` to the available parallelism instead of 8.
+## 4.1.0 [2023-01-24]
 
-## 4.0.0 (2023-10-31)
+- Default the concurrency of the `ThreadedExecutor` to the available parallelism instead of 8.
+- Support custom syscalls (only needed for non-Filecoin users).
+    - Filecoin now uses the `FilecoinKernel`, not the `DefaultKernel`. The `DefaultKernel` no longer implements Filecoin specific features.
+    - The `Kernel` trait now inherits from the `SyscallHandler` trait, allowing kernel's to decide how they want to expose themselves to actors via syscalls.
+- Set the `ThreadedExecutor`'s thread-pool size equal to the number of available system threads.
```

Comment:
You already said `Default the concurrency of the `ThreadedExecutor` to the available parallelism instead of 8.` above?